exercises/change/cases_test.go

```diff
@@ -0,0 +1,138 @@
+package change
+
+// Source: exercism/x-common
+// Commit: cda8f98 Create new exercises structure
```

Comment:
Looks like interface{} is my friend to overcome the [list] vs. -1 difference.  Pursuing that.